site/docs/latest/admin-api/namespaces.md

```diff
@@ -344,7 +344,7 @@ N/A
 ###### REST
 
 ```
-POST /admin/persistent/{tenant}/{namespace}/persistence
+POST /admin/v2/persistent/{tenant}/{namespace}/persistence
```

Comment:
Here the doc was wrong in that the path should be `/admin/v2/namespaces/..` instead of `persistent/...`